cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -415,12 +420,17 @@ message InstalledPackageSummary {
   // is installed, then:
   //   * if 10.3.2 is available, latest_matching_version should be 10.3.2, but
   //   * if 10.4 is available while >10.3.1 is not, this should remain empty.
-  string latest_matching_pkg_version = 8;
+  string latest_matching_pkg_version = 9;
 
   // LatestPkgVersion
   //
   // The latest version available for this package, regardless of the pkg_version_reference.
-  string latest_pkg_version = 9;
+  string latest_pkg_version = 10;
+
+  // Status
+  //
+  // The current status of the installed package.
+  InstalledPackageStatus status = 11;
```

Comment:
So, right now we should go with the polling approach, don't we?
If I understood it properly. the alternative to our current ws impl is using grpc streams like:

`rpc getInstalledPackageSummaryStatus(InstalledPackageSummaryRequest) returns (stream Status);` 

?  (but totally out of scope right now, I know)